sdk/eventhub/azure-eventhub/tests/conftest.py

```diff
@@ -110,100 +112,73 @@ def timeout_factor(uamqp_transport):
 def fake_span():
     return FakeSpan
 
+@pytest.fixture()
+def enable_tracing(fake_span):
+    settings.tracing_implementation.set_value(fake_span)
+    yield fake_span
+    # MUST RESET TRACING IMPLEMENTATION TO NONE, ELSE ALL OTHER TESTS ADD TRACING
+    settings.tracing_implementation.set_value(None)
 
 @pytest.fixture(scope="session")
 def get_credential():
-    use_pwsh = os.environ.get("AZURE_TEST_USE_PWSH_AUTH", "false")
-    use_cli = os.environ.get("AZURE_TEST_USE_CLI_AUTH", "false")
-
-    # User-based authentication through Azure PowerShell, if requested
-    if use_pwsh.lower() == "true":
-        log.info(
-            "Environment variable AZURE_TEST_USE_PWSH_AUTH set to 'true'. Using AzurePowerShellCredential."
-        )
-        from azure.identity import AzurePowerShellCredential
-        return AzurePowerShellCredential()
-    # User-based authentication through Azure CLI, if requested
-    elif use_cli.lower() == "true":
-        log.info("Environment variable AZURE_TEST_USE_CLI_AUTH set to 'true'. Using AzureCliCredential.")
-        from azure.identity import AzureCliCredential
-        return AzureCliCredential()
-    return EnvironmentCredential()
+    return get_devtools_credential
```

Comment:
nit - but could this just yield back the credential itself, so in the tests you can just expect the credential back?

Reply:
I tried that before, but something about creating it a little earlier seemed to have been messing up timing in the tests, so that it expired/auth wouldn't always work when created in the fixture.